net/HttpRequest.hpp

```diff
@@ -441,6 +464,30 @@ class Header
     /// Return true iff Transfer-Encoding is set to chunked (the last entry).
     bool getChunkedTransferEncoding() const { return _chunked; }
 
+    bool hasConnectionToken() const { return has(CONNECTION); }
+    ConnectionToken getConnectionToken() const {
+        const std::string token = get(CONNECTION);
+        if( Util::iequal("close", token) ) {
+            return ConnectionToken::Close;
+        } else if( Util::iequal("keep-alive", token) ) {
```

Comment:
Ditto as above. No need for all these `else if` after returning. Just have these as `if/return` cases, without all the `else`s. More readable as the structure of the code reflects the intent and functionality.